src/main/java/com/fasterxml/jackson/databind/cfg/MutableConfigOverride.java

```diff
@@ -41,6 +41,11 @@ public MutableConfigOverride setInclude(JsonInclude.Value v) {
         return this;
     }
 
+    public MutableConfigOverride setIncludeAsProperty(JsonInclude.Value v) {
```

Comment:
Would be nice to add javadocs (even if other methods don't necessarily have them either :) ) for brief explanation.